examples/integers/src/main.pl

```diff
@@ -0,0 +1,13 @@
+use strict;
+use warnings;
+use 5.010;
```

Comment:
Most of the examples require Perl 5.10 feature of `say` which makes the code more like the other languages and easier to read without having to us `print` and explicitly put an `"\n"`

Reply:
+1